index.html

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./styles.css" />
+    <title>Exquisite Residence</title>
+    <script
+      src="https://kit.fontawesome.com/e1fe8e6842.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <nav>
+      <ul class="icon">
+        <li>
+          <a href="#"

```

Comment:
- [ ] Please apply some background color for the hamburger menu and reduce the transparency, Right now the menu is more transparent than it should be, and the menu content and the main content area are visible together and it's hard for users to read the mobile menu content. Fixing this will your project looks more professional and will also improve the user experience.

### See the image for reference 

![image](https://user-images.githubusercontent.com/83803414/169280170-a2cd5f13-3e6e-45a9-92fe-01c17ed7ace1.png)